src/NzbDrone.Core/MetadataSource/SkyHook/SkyHookProxy.cs

```diff
@@ -93,11 +84,30 @@ public Artist GetArtistInfo(string foreignArtistId, int metadataProfileId)
             artist.Metadata = MapArtistMetadata(httpResponse.Resource);
             artist.CleanName = Parser.Parser.CleanArtistName(artist.Metadata.Value.Name);
             artist.SortName = Parser.Parser.NormalizeTitle(artist.Metadata.Value.Name);
-            artist.Albums = httpResponse.Resource.Albums.Select(x => MapAlbum(x, null)).ToList();
+
+            artist.Albums = FilterAlbums(httpResponse.Resource.Albums, metadataProfileId)
+                .Select(x => new Album {
+                        ForeignAlbumId = x.Id
+                    })
+                .ToList();
 
             return artist;
         }
 
+        public IEnumerable<AlbumResource> FilterAlbums(IEnumerable<AlbumResource> albums, int metadataProfileId)
```

Comment:
Do we really want to make these public just for testing sake? Given the skyhook tests are classified as integration tests could we just run a getartistinfo in test fixture and validate albums were filtered instead of calling function direct?